Test/Get-RivetConfig.Tests.ps1

```diff
@@ -480,4 +494,41 @@ Describe 'Get-RivetConfig.when databases have a custom order' {
         $config.Databases[2].Name | Should -Be 'AAA'
         $config.Databases[3].Name | Should -Be 'DDD'
     }
+}
+
+Describe 'Get-RivetConfig.when plugins root has a wildcard' {
+    It 'should resolve to actual path' {
+        Init
+        GivenDatabase 'Config'
+        GivenDirectory 'Extensions\0.1.0\Plugins'
+        GivenConfig @'
+{
+    "SqlServerName": ".\\Rivet",
+    "DatabasesRoot": "Databases",
+    "PluginsRoot": "Extensions\\*\\Plugins"

```

Comment:
Should we add a test for the other types of PowerShell wildcards? E.g. single character `0.1.?` or sets/ranges `0.[12].*` ?

Reply:
If we were handling the wildcard matching, I would say yes. But since that is handled by `Resolve-Path`, I think one test is sufficient.